torch/_torch_docs.py

```diff
@@ -3161,7 +3161,7 @@
 Indices are ordered from left to right according to when each was sampled
 (first samples are placed in first column).
 
-If :attr:`input` is a vector, :attr:`out` is a vector of size :math:`num_samples`.
+If :attr:`input` is a vector, :attr:`out` is a vector of size :math:`num\_samples`.
```

Comment:
I think we just want this to be an `:attr:` instead of using the math mode. This will render every letter like a variable

Reply:
I agree. ``:attr:`num_samples` `` would be great.